modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeVelocityLiferayTaglibReferenceMigrationCheck.java

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.CharPool;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringBundler;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.constants.VelocityMigrationConstants;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * @author Nícolas Moura
+ */
+public class UpgradeVelocityLiferayTaglibReferenceMigrationCheck
+	extends BaseUpgradeVelocityMigrationCheck {
+
+	@Override
+	protected String migrateContent(String content) {
+		String[] lines = StringUtil.splitLines(content);
+
+		for (int i = 0; i < lines.length; i++) {
+			String newLine = lines[i];
+
+			if (newLine.contains(_VELOCITY_LIFERAY_LANGUAGE_FORMAT)) {
+				String[] possibilities = {
+					_VELOCITY_LIFERAY_LANGUAGE_FORMAT +
+						_VELOCITY_LIFERAY_LANGUAGE_FORMAT_ARGUMENTS,
+					_VELOCITY_LIFERAY_LANGUAGE_FORMAT + CharPool.SPACE +
+						_VELOCITY_LIFERAY_LANGUAGE_FORMAT_ARGUMENTS
+				};
+
+				lines[i] = StringUtil.replace(
+					newLine, possibilities,
+					new String[] {
+						_FREEMARKER_LIFERAY_LANGUAGE_FORMAT,
+						_FREEMARKER_LIFERAY_LANGUAGE_FORMAT
+					});
```

Comment:
@nicolas-sss, this code seems to be a pattern, can we try to create a new private method to hold this logic and reuse? Does it make sense to you?